src/software/sensor_fusion/sensor_fusion_test.cpp

```diff
@@ -194,35 +221,89 @@ TEST_F(SensorFusionTest, test_complete_wrapper_with_tbots_robot_msg_1_at_a_time)
     SensorMsg sensor_msg_1;
     auto ssl_wrapper_packet =
         createWrapperPacket(std::move(geom_data), std::move(detection_frame));
-    *(sensor_msg_1.mutable_ssl_vision_msg()) = *(ssl_wrapper_packet.release());
-    *(sensor_msg_1.add_tbots_robot_msgs())   = *(tbots_robot_msg_id_1.release());
+    *(sensor_msg_1.mutable_ssl_vision_msg()) = *ssl_wrapper_packet;
+    *(sensor_msg_1.add_tbots_robot_msgs())   = *tbots_robot_msg_id_1;
```

Comment:
are we leaking memory here? is this what its fixing?

Im worried adding msgs to repeated proto fields is kind of confusing and it places a burden on future members to understand whats going on, is there something we can do (like a generic helper of some sorts) to prevent mistakes when adding to a `add_tbots_robot_msgs`?

out of scope for this ticket, but just a thought

Reply:
it's confusing that we have different conventions for repeated and non-repeated fields. How can we unify them?

Reply:
Just so we're all on the same page, what are the conventions for both?

Reply:
After offline discussion, @akhilveeraghanta and I agreed that we should use `*(sensor_msg_1.mutable_ssl_vision_msg()) = *ssl_wrapper_packet;` style. See #1557 for more details